test/test_functional_box.py

```diff
@@ -355,9 +355,6 @@ def test_camel_killer_box(self):
         assert con_kill_box.camel_case == 'Item'
         assert con_kill_box.x321_camel_case_fever == 'Safe'
 
-        with pytest.raises(BoxKeyError):
```

Comment:
I wasn't able to quickly solve this issue, so in the meanwhile I removed the regression test. Someone (maybe me) may revert this specific commit in the future to get the regression test.